packages/noya-state/src/reducers/pointReducer.ts

```diff
@@ -41,6 +48,23 @@ export function pointReducer(
         curvePoint.cornerRadius = Math.max(0, newValue);
       });
     }
+    case 'selectPoint': {
+      const [, selectedPoint, selectionType] = action;
```

Comment:
For consistency with other selection actions, and to simplify the loop, lets assign a default value to `selectionType` here: `selectionType = 'replace'`